hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/SystemFileUpdateFacility.java

```diff
@@ -98,49 +105,51 @@ public void handleTxBody(@NonNull final HederaState state, @NonNull final Transa
         final var fileNum = fileID.fileNum();
         final var payer = txBody.transactionIDOrThrow().accountIDOrThrow();
         if (fileNum > ledgerConfig.numReservedSystemEntities()) {
-            return;
+            return SUCCESS;
         }
 
         // If it is a special file, call the updater.
         // We load the file only, if there is an updater for it.
         final var config = configuration.getConfigData(FilesConfig.class);
-        try {
-            if (fileNum == config.addressBook()) {
-                logger.error("Update of address book not implemented");
-            } else if (fileNum == config.nodeDetails()) {
-                logger.error("Update of node details not implemented");
-            } else if (fileNum == config.feeSchedules()) {
-                logger.error("Update of fee schedules not implemented");
-            } else if (fileNum == config.exchangeRates()) {
-                exchangeRateManager.update(FileUtilities.getFileContent(state, fileID), payer);
-            } else if (fileNum == config.networkProperties()) {
-                configProvider.update(FileUtilities.getFileContent(state, fileID));
-                backendThrottle.applyGasConfig();
-                frontendThrottle.applyGasConfig();
 
-                // Updating the multiplier source to use the new gas throttle
-                // values that are coming from the network properties
-                congestionMultipliers.resetExpectations();
-            } else if (fileNum == config.hapiPermissions()) {
-                logger.error("Update of HAPI permissions not implemented");
-            } else if (fileNum == config.throttleDefinitions()) {
-                throttleManager.update(FileUtilities.getFileContent(state, fileID));
-                backendThrottle.rebuildFor(throttleManager.throttleDefinitions());
-                frontendThrottle.rebuildFor(throttleManager.throttleDefinitions());
+        if (fileNum == config.feeSchedules()) {
```

Comment:
Is there is an issue to add address book and node details support at some point?

Reply:
We don't have to do anything with the `addressBook` or `nodeDetails`. I did not know this when I wrote the placeholders above.